CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Added
 
+- Self-hosted etcd will mount a hostPath volume directory defined as
+  value of annotation `selfhosted-etcd.alpha.coreos.com/checkpointdir`.
```

Comment:
@aaronlevy @diegs @xiang90 

We will use this annotation in cluster TPR to specify the dir that contains checkpoints. Currently it is `/etc/kubernetes/checkpoint-iptables` in bootkube.